docs/DeveloperGuide.md

```diff
@@ -1040,6 +1040,34 @@ testers are expected to do more *exploratory* testing.
 
 3. _{ more test cases … }_
 
+### Rooms
+
+1. Adding a room
+
+    1. Prerequisites: There are no rooms in SunRez, or the room with the number `10-999` does not exist in the system.
+
+    2. Test case: `oadd r/10-999 t/suite_ac` <br>
+       Expected: A feedback message in the result box indicating the room was successfully added. The room should also be visible in the room panel.
+
+    3. Test case: `oadd r/10-999 t/suite_ac` <br>
+       Expected: An error message indicating the room `10-999` already exists in the system.
```

Comment:
Maybe indicate that this test case continues from step 2.

Reply:
Done